### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by rachit77**
Tests were failing because of bad kurtosis config in workflow

### CODE REVIEW

File: .github/workflows/regression-tests.yml [hunk 1]
```diff
@@ -19,25 +30,35 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: 0xPolygon/kurtosis-cdk
-          ref: main
+          ref: "v0.2.15"
```
Comment (Stefan-Ethernal) [inline .github/workflows/regression-tests.yml:33]: Why not use the `v0.2.22`?
https://github.com/0xPolygon/kurtosis-cdk/releases/tag/v0.2.22

File: .github/workflows/regression-tests.yml [hunk 2]
```diff
@@ -19,25 +30,35 @@ jobs:
...
 
-      - name: Install Kurtosis CDK tools
-        uses: ./kurtosis-cdk/.github/actions/setup-kurtosis-cdk
+      - name: Install kurtosis
+        shell: bash
+        run: |
+          echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+          sudo apt update
+          sudo apt install kurtosis-cli=1.3.0
+          kurtosis version
```
Comment (Stefan-Ethernal) [inline .github/workflows/regression-tests.yml:42]: IMO it is better to stick with the GH action (it makes it more streamlined).

File: .github/workflows/regression-tests.yml [hunk 3]
```diff
@@ -19,25 +30,35 @@ jobs:
...
-        working-directory: ./kurtosis-cdk
-        run: |
-          yq -Y --in-place '.args.data_availability_mode = "cdk-validium"' params.yml
-          yq -Y --in-place '.args.zkevm_da_image = "cdk-data-availability:local"' params.yml
-
       - name: Deploy Kurtosis CDK package
         working-directory: ./kurtosis-cdk
-        run: kurtosis run --enclave cdk-v1 --args-file params.yml --image-download always .
+        run: >
+          kurtosis run --enclave cdk . '{"args": {"zkevm_da_image": "cdk-data-availability:local"}}'
```
Comment (Stefan-Ethernal) [inline .github/workflows/regression-tests.yml:61]: We need to provide the `consensus_contract_type` variable as well (https://github.com/0xPolygon/kurtosis-cdk/blob/473422d4a02d978de2b61199e444fd7d0c98f7f6/input_parser.star#L297). It should be the `cdk-validium`.

**Review by Stefan-Ethernal — COMMENTED** [review]
As a side note:
we are not generating any load to the CDK node, so empty batches are created?

Probably we should consider using this script to generate some load and make sure batches are in verified state: https://github.com/0xPolygon/kurtosis-cdk/blob/main/.github/scripts/monitor-verified-batches.sh. WDYT?

File: .github/workflows/regression-tests.yml (hunk 3 above)
Comment (rachit77) [reply .github/workflows/regression-tests.yml:61]: isn't it already `cdk-validium` by default?

File: .github/workflows/regression-tests.yml (hunk 2 above)
Comment (rachit77) [reply .github/workflows/regression-tests.yml:42]: will use that action from kurtosis.
On a side note we don't use the setup action in cdk monorepo

File: .github/workflows/regression-tests.yml (hunk 3 above)
Comment (Stefan-Ethernal) [reply .github/workflows/regression-tests.yml:61]: Yes, but I was thinking, maybe it is better to be explicit (in case that default value changes). WDYT?

**Review by Stefan-Ethernal — APPROVED** [review]

**Pull request merged**